app/src/main/java/com/moviefiend/torian/moviefiend/MovieViewHolder.java

```diff
@@ -0,0 +1,39 @@
+package com.moviefiend.torian.moviefiend;
+
+import android.support.v7.widget.RecyclerView;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.squareup.picasso.Picasso;
+
+public class MovieViewHolder extends RecyclerView.ViewHolder {
```

Comment:
@ifeins critical? especially now that the view holder has some "logic" I rather keep it clean